### PR TITLE
[3563][MIG] base_technical_features_adj: Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3563](https://www.quartile.co/web#view_type=form&model=project.task&id=3563&active_id=3563&menu_id=)

### CODE REVIEW

**Review by yostashiro — COMMENTED** [review]
I think there was miscommunication/misunderstanding.  My suggestion was to appy this adjustment directly to the base_technical_features module (create a PR in OCA, of course) instead of creating a separate module.

**Comment by AungKoKoLin1997** [comment]
> I think there was miscommunication/misunderstanding. My suggestion was to appy this adjustment directly to the base_technical_features module (create a PR in OCA, of course) instead of creating a separate module.

@yostashiro In standard operation, non internal users don't have backend access and this adjustment is not required. I think this is our specific need and we should keep in our repo, not in OCA.

**Comment by yostashiro** [comment]
@AungKoKoLin1997 As you know there is a PR of base_group_backend and it makes the update on the original OCA module legitimate, IMO.  Of course we should explain this as the background when we create a PR in the OCA.

**Comment by yostashiro** [comment]
On second thought, we may just go with this PR if there can be some overhead with updating the original OCA module.

File: base_technical_features_adj/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module limits the visibility of the 'technical_features' attribute
+in the preference view, making it accessible exclusively to internal users.
```
Comment (yostashiro) [inline base_technical_features_adj/readme/DESCRIPTION.rst:2]: Please add the background information.

File: base_technical_features_adj/README.rst [hunk 2]
```diff
@@ -22,6 +22,15 @@ Technical features group adjustments
 This module limits the visibility of the 'technical_features' attribute
 in the preference view, making it accessible exclusively to internal users.
 
+Background
+~~~~~~~~~~
+
+In the SST environment, backend access is currently allowed for non-internal users.
```
Comment (yostashiro) [inline base_technical_features_adj/README.rst]: @AungKoKoLin1997 Avoid making the module description customer specific, especially the ones in public repos.  This explanation can be much shorter as well (avoid verbosity).

**Review by yostashiro — APPROVED** [review]

**Comment by yostashiro** [comment]
@AungKoKoLin1997 Please squash your commits.

**Review by kanda999 — APPROVED** [review]

**Pull request merged**